docs/solid-remotestorage.md

```diff
@@ -0,0 +1,27 @@
+## Solid remote storage development
+ In order to start development:
+```sh
+./init/solid-remote-storage.sh
+ ```
+This will create a folder named `remotestorage-dev`. There will be cloned directories for repositories [remotestorage.js](https://github.com/pondersource/remotestorage.js) and [remotestorage-widget](https://github.com/pondersource/remotestorage-widget/) in folders respectively called `remotestorage` and `remotestorage-widget`.
+
+After init you have to run the server by using this command:
+```sh
+./dev/solid-remote-storage.sh
+```
+And a server will be listening to port `8080`.
+
+After making a change to each of the remotstorage.js or remotestorage-widget repositories, make sure to call:
+```sh
+./scripts/solid-rebuild-remote-storage.sh
```

Comment:
you can use npm run dev to avoid this step each time you change the source code
it watches the source code using the config file and recompiles it.

https://github.com/pondersource/remotestorage.js/blob/3abd9a13209b38296ba3f021280b0051af79d0cc/package.json#L27C9-L27C9

Reply:
done in this commit https://github.com/pondersource/dev-stock/pull/60/commits/f95e6d97ad9c88e5b09c9ac2a2e641863848106c